docs/api.md

```diff
@@ -616,8 +617,6 @@ var sync = PouchDB.sync('mydb', 'http://localhost:5984/mydb', {live: true})
     // handle change
   }).on('complete', function (info) {
     // handle complete
-  }).on('uptodate', function (info) {
-    // handle up-to-date
   }).on('error', function (err) {
```

Comment:
OK, in that case +1, but yeah, we need more docs.

Probably also a big blog post introducing the new feature, and also a new section in the guide.